monailabel/transform/pre.py

```diff
@@ -60,3 +93,129 @@ def __call__(self, data):
             label[label > 0] = self.value
             d[key].array = label
         return d
+
+
+class SuppressStderr:
```

Comment:
MONAI already has this class:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/data/video_dataset.py#L33

Thanks.